makefile

```diff
@@ -0,0 +1,26 @@
+
+APP_NAME := j1708-tester
+BUILD_PATH := ./artifacts
+LINUX_BUILD_PATH = $(BUILD_PATH)/linux/$(APP_NAME)
+LINUX_ARM_BUILD_PATH = $(BUILD_PATH)/arm/$(APP_NAME)
+WINDOWS_BUILD_PATH = $(BUILD_PATH)/windows/$(APP_NAME).exe
+MAC_BUILD_PATH = $(BUILD_PATH)/darwin/$(APP_NAME)
+
+.PHONY: build run package
```

Comment:
Is this combination of default targets intended? Seems like an odd order.

Reply:
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

Reply:
Ahhhhh, I get it now. Thanks